utils/ci/corefile_in_gha.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+cd daos || exit 1
+
+echo ::group::Check for corefiles
+
+echo BASE_DISTRO=${BASE_DISTRO}
+echo -n core_pattern= ; cat /proc/sys/kernel/core_pattern
+
+if [ $(echo "$BASE_DISTRO" | egrep -i 'ubuntu|alma|rocky') ]
```

Comment:
(lint) Quote this to prevent word splitting. [SC2046]
(lint) Use egrep -q instead of comparing output with [ -n .. ]. [SC2143]
(lint) egrep is non-standard and deprecated. Use grep -E instead. [SC2196]